### PR TITLE
Added snapshot to compare results with snapshot tests and also update…

### PR DESCRIPTION
**Pull request by ajitava-git**
This PR includes a snapshot that was created using pytest-snapshot 0.9.0
The created snapshot can now be compared with the snapshot tests

Requirements.txt has also been updated with pytest, csv23, requests and pytest-snapshot.

### CODE REVIEW

File: requirements.txt
Comment (avinash010) [inline requirements.txt:1]: Why do we need pytest in the main requirements.txt file? Do we need this to run the practice testing aiml app?

Comment (ajitava-git) [reply requirements.txt:1]: @avinash010 We don't. This was a part of when I did the initial testing by doing the changes in the root requirements file. As we have moved everything under the tests/requirements file, we no longer need it. I have removed it.

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Flask==2.2.2
```
Comment (avinash010) [inline requirements.txt:1]: @ajitava-git Also sklearn is being removed. We no longer need it?

File: tests/test_snapshot_is_pto_score.py [hunk 2]
```diff
@@ -87,4 +89,3 @@ def test_snapshot_accuracy(snapshot):
 
     #Creating snapshot directory
     snapshot.assert_match(f"{accuracy},{precision},{recall},{f1_score}","overall_score.txt")
```
Comment (avinash010) [inline tests/test_snapshot_is_pto_score.py:91]: @ajitava-git Did we run this code through pylint?
I see some changes to the file formatting

File: requirements.txt (hunk 1 above)
Comment (ajitava-git) [reply requirements.txt:1]: Yes @avinash010 . sklearn was obsolete.

File: tests/test_snapshot_is_pto_score.py (hunk 2 above)
Comment (ajitava-git) [reply tests/test_snapshot_is_pto_score.py:91]: @avinash010 Yes, ran it through pylint.

**Pull request merged**